hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1895,7 +1899,7 @@ func (r *HostedClusterReconciler) reconcileIgnitionServer(ctx context.Context, c
 // reconcileAutoscaler orchestrates reconciliation of autoscaler components using
 // both the HostedCluster and the HostedControlPlane which the autoscaler takes
 // inputs from.
-func (r *HostedClusterReconciler) reconcileAutoscaler(ctx context.Context, createOrUpdate upsert.CreateOrUpdateFN, hcluster *hyperv1.HostedCluster, hcp *hyperv1.HostedControlPlane) error {
+func (r *HostedClusterReconciler) reconcileAutoscaler(ctx context.Context, createOrUpdate upsert.CreateOrUpdateFN, hcluster *hyperv1.HostedCluster, hcp *hyperv1.HostedControlPlane, image string) error {
```

Comment:
we also would need:
https://github.com/openshift/hypershift/blob/c276df9e1d6d5bddb8bd72de650e68470e5816d7/hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go#L2842

removed

Reply:
this is set in the deployment reconciler already, rigth?